### DISCUSSION TITLE
How do you checkout a tag?

### DISCUSSION THREAD

**Discussion by acampove** [NeogitOrg/neogit#1547] [Q&A]
I am trying to do:

```
git checkout v1
```

where `v1` is a tag. But I do not see that in the Branch:

![image](https://github.com/user-attachments/assets/3f4adf91-5f1c-42d2-9654-8143499bc0be)

or tag 

![image](https://github.com/user-attachments/assets/085589c3-35a0-4e30-a89a-c13d731dbb04)

menus

**Comment by CKolkey** [comment]
You can see a list of refs by pressing `y`. Find the tag you want to check out, and with your cursor over it, press `bb` - it should be the first item on the list.

Works the same from the log view, btw

**Reply by acampove** [reply]
Hi @CKolkey, thanks for your reply and for the project. I just realized that with `bb` I do not just get a list of the branches but the tags are also included. However with `y`, as you said, I can see a separate section showing only the tags.
